docs/spec.yaml

```diff
@@ -0,0 +1,357 @@
+        '201':
+          description: Created
+          headers:
+            Location:
+              description: 'Location of the Client instance on the proxy server'
+              schema:
+                type: string
+              example:
+                Location: 'client/464bb685-fd58-41b6-afa0-6df49e38c705'
+          content:
+            application/json:
+              schema: 
+                type: object
+                properties:
+                  message:
+                    type: string
+                    description: 'A message indicating the result of initializing an SDK client'
+                required:
+                  - message
+              example:
+                  message: 'Success'
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema: 
+                type: object
+                properties:
+                  exception:
+                    type: string
+                    description: 'An error from the SDK indicating an exception was thrown during initialization'
+              example:
+                exception: 'Missing environment key! Call initialize with a valid environment key'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema: 
+                type: object
+                properties:
+                  message:
+                    type: string
+                    description: 'An error message indicating what went wrong with the request'
+                required: 
+                  -  message
+              example:
+                message: 'Invalid request: missing clientId'
+
+  '/{location}/command':
+    parameters: 
+      - name: location
+        in: path
+        required: true
+        description: 'Location of the instance that the method is being called on. For example, `client/464bb685-fd58-41b6-afa0-6df49e38c705` or `variable/3'
+        schema: 
+          type: string
+    post:
+      tags:
+        -  Location
+      
+      summary: Invoke a method on a previously created instance
+      operationId: command
+      security: []
+      
+      requestBody:
+        $ref: '#/components/requestBodies/ClientCommandRequestBody'
+      
+      responses:
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema: 
+                properties:
+                  entityType:
+                    type: string
+                    description: 'Type of entity returned. One of `Variable`, `User`, `Feature`, `Object` or `Void`'
+                  data:
+                    type: object
+                    description: 'Value returned by the method'
+                  logs:
+                    type: array
+                    description: 'Logs generated during the method invocation'
+                    items:
+                      type: string
+                  exception:
```

Comment:
this should be under a "200" response. It's only 201 if the command succeeded

Reply:
yup, made the change